src/topp/FileConverter.cpp

```diff
@@ -460,6 +463,12 @@ class TOPPFileConverter :
 
     if (out_type == FileTypes::MZML)
     {
+      if (exp == empty_exp)
+      {
+        OPENMS_LOG_ERROR << "No input data: no MS1/MS2 data present! Cannot write mzML. Please use another input/output format combination.";
```

Comment:
does this include files just with chromatograms?

Reply:
yes. even metadata would be fine and produce an output.
(note: `if (exp.empty())` does not work since it only checks for spectra)